articles/multifactor-authentication/overview.md

```diff
@@ -0,0 +1,110 @@
+
+![MFA Dashboard Page](/media/articles/multifactor-authentication/mfa-dashboard.png)
+
+Any or all of these factors can be enabled simultaneously. When logging in the first time, the user will be shown a default and but will be allowed to choose another factor to use, if you have more than one factor enabled in the Dashboard. 
+
+**TODO: IMAGE OF CURRENT LOGIN PROMPT WITH MFA CHOICES**
+
+::: note
+When you enable the Duo factor, you will have to fill in a few further settings related specifically to Duo before continuing.
+**TODO: SPECIFY SETTINGS**
+:::
+
+Additionally, there is the **Always require Multi-factor Authentication** toggle, which allows authentication against this app only when one or more of the factors is enabled in the Dashboard, and completed by users while logging in.
+
+### 2. Set up your services
+
+Auth0 supports the following factors for implementing MFA. You must enable at least one to use MFA, but you can choose to enable and make available more than one factor if you wish.
+
+#### One Time Passwords (OTP)
+
+The principle behind OTP as a factor is fairly straightforward for the end user. They use an app, such as:
+
+* Authy ([Google Play](https://play.google.com/store/apps/details?id=com.authy.authy) / [App Store](https://itunes.apple.com/us/app/authy/id494168017)).
+* Google Authenticator ([Google Play](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) / [App Store](https://itunes.apple.com/us/app/google-authenticator/id388497605)).
+* Guardian ([Google Play](https://play.google.com/store/apps/details?id=com.auth0.guardian) / [App Store](https://itunes.apple.com/us/app/auth0-guardian/id1093447833)).
+* Microsoft Authenticator ([Google Play](https://play.google.com/store/apps/details?id=com.azure.authenticator) / [App Store](https://itunes.apple.com/us/app/microsoft-authenticator/id983156458)).
+
+Upon signup, they can scan a code and set up the app, upon which it will begin generating one-time codes. Afterword, the user can simply check the app for the current one-time code to enter when authenticating using this factor.
+
+Your users will need to have in their posession a supported device for whichever OTP app you use.
+
+#### Push notifications
+
+MFA by push notification simply requires an appropriate app on the user's device. The app is sent push notifications when the user attempts to authenticate, and the user must respond to it in order to login, ensuring that they not only know their login information but also posess the device set up for MFA.
+
+End users will be prompted to download whichever app you have enabled in the Dashboard when trying to sign up or log in to your application. Once they indicate that they have successfully downloaded the app, a barcode will appear on screen. They will have a short amount of time in which to scan the code with the designated app. Once this is done, they should see a confirmation screen and also a recovery code to take note of.
```

Comment:
> will be prompted to download whichever app you have enabled in the Dashboard
You can't enable a specific App on the dashboard now (DUO uses a totally different UI altogether), it will always be the Guardian App. You'll have two buttons to download either Android or iOS apps.